### PR TITLE
first version of creating tables

### PR DESCRIPTION
**Pull request by xiaoqingxing**
script tested on MYSQL 5.7.29

### CODE REVIEW

**Comment by xiaoqingxing** [comment]
Okay, I'll make a new push request.

File: mysql_scripts/4ncov_service_mysql.sql [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+DROP TABLE IF EXISTS `4ncov`.`user_role` ;
+
+CREATE TABLE IF NOT EXISTS `4ncov`.`user_role` (
+  `id` INT UNSIGNED NOT NULL AUTO_INCREMENT COMMENT '角色ID，目前应该有：\n\n1. 管理员\n2. 供应商\n3. 需求方',
+  `user_role_desc` VARCHAR(45) NOT NULL COMMENT '角色描述',
+  `gmt_created` DATETIME NOT NULL,
+  `gmt_modified` DATETIME NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8
```
[the remark this answers is not in the document]
Comment (lrcry) [reply mysql_scripts/4ncov_service_mysql.sql]: Has created the database with `utf8mb4`, no need to set charset upon table creation

File: mysql_scripts/4ncov_service_mysql.sql [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+CREATE SCHEMA IF NOT EXISTS `4ncov` ;
+USE `4ncov` ;
+
+-- -----------------------------------------------------
+-- Table `4ncov`.`user_role`
+-- -----------------------------------------------------
+DROP TABLE IF EXISTS `4ncov`.`user_role` ;
+
+CREATE TABLE IF NOT EXISTS `4ncov`.`user_role` (
+  `id` INT UNSIGNED NOT NULL AUTO_INCREMENT COMMENT '角色ID，目前应该有：\n\n1. 管理员\n2. 供应商\n3. 需求方',
```
[the remark this answers is not in the document]
Comment (lrcry) [reply mysql_scripts/4ncov_service_mysql.sql]: Please see https://github.com/4ncov/service-4ncov#%E5%BC%80%E5%8F%91%E5%8E%9F%E5%88%99

**Comment by lrcry** [comment]
#14

**Comment by lrcry** [comment]
Closes #14

**Pull request merged**